src/services/WolfDen.Application/Requests/Queries/Employees/EmployeeDirectory/GetAllEmployeeQuery.cs

```diff
@@ -4,14 +4,18 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-using WolfDen.Application.DTOs;
+using WolfDen.Application.DTOs.Employees;
 
 namespace WolfDen.Application.Requests.Queries.Employees.EmployeeDirectory
 {
-    public class GetAllEmployeeQuery : IRequest<List<EmployeeDirectoryDTO>>
+    public class GetAllEmployeeQuery : IRequest<EmployeeDirecotyWithPageCountDTO>
     {
         public int? DepartmentID { get; set; }
         public string? EmployeeName {  get; set; }
+        public int PageNumber {  get; set; }
+
+        public int PageSize { get; set; }
```

Comment:
You can use a common wrapper class `PaginationRequest` for common pagination parameters